src/client/powerpages/Contentsnippet.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+import * as nls from "vscode-nls";
+nls.config({
+    messageFormat: nls.MessageFormat.bundle,
+    bundleFormat: nls.BundleFormat.standalone,
+})();
+const localize: nls.LocalizeFunc = nls.loadMessageBundle();
+import * as vscode from "vscode";
+import { fileName, folderName, isNullOrEmpty } from "./utils/CommonUtils";
+import { QuickPickItem } from "vscode";
+import { MultiStepInput } from "./utils/MultiStepInput";
+
+export const contentSnippet = async (context: vscode.ExtensionContext) => {
+    const { name, type } = await myMultiStepInput();
+
+    if (!isNullOrEmpty(name)) {
+        const terminal = vscode.window.createTerminal("Powerpages", "");
```

Comment:
Should'nt this be localized?

Reply:
I thought proper nouns are not localized, do we localize them also?